### PR TITLE
Fixed numerous broken links.

### LINKED ISSUE #1472: Broken Link (enroll-hybrid-ios-mac.md)
In the file:
SCCMdocs\sccm\mdm\deploy-use\enroll-hybrid-ios-mac.md

The following link is broken:
http://go.microsoft.com/fwlink/?LinkId=269844

This links to an Apple site with a specific key, so I'm not sure what it'd be replaced with.

(admin edit, add metadata for article link)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 36fa2425-75b6-781f-1656-92ce0ba456c3
* Version Independent ID: f84126e3-c02e-be32-dc74-da3b8b2d8889
* Content: [Set up iOS and Mac hybrid device management with Microsoft Intune - Configuration Manager](https://docs.microsoft.com/en-us/sccm/mdm/deploy-use/enroll-hybrid-ios-mac)
* Content Source: [sccm/mdm/deploy-use/enroll-hybrid-ios-mac.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/mdm/deploy-use/enroll-hybrid-ios-mac.md)
* Product: **configuration-manager**
* Technology: **configmgr-hybrid**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### LINKED ISSUE #1476: Broken Link (privacy-statement.md)
In the file:
SCCMdocs\sccm\core\misc\privacy\privacy-statement.md

The following link is broken:
http://go.microsoft.com/fwlink/p/?LinkId=224911

### LINKED ISSUE #1477: Broken Link (manage-settings-for-software-updates.md)
In File:
SCCMdocs\sccm\sum\get-started\manage-settings-for-software-updates.md

The following link is broken:
http://go.microsoft.com/fwlink/p/?LinkId=58313

I could not find a replacement link.

(admin edit: add metadata for article link)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 54216432-726f-0fbb-5001-1c1021320ba9
* Version Independent ID: b39f5e74-541d-de5b-4a93-77f9627775f5
* Content: [Manage settings for software updates - Configuration Manager](https://docs.microsoft.com/en-us/sccm/sum/get-started/manage-settings-for-software-updates)
* Content Source: [sccm/sum/get-started/manage-settings-for-software-updates.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/sum/get-started/manage-settings-for-software-updates.md)
* Product: **configuration-manager**
* Technology: **configmgr-sum**
* GitHub Login: @mestew
* Microsoft Alias: **mstewart**

### LINKED ISSUE #1478: Broken Link (samples-guide.md)
In File:
SCCMdocs\sccm\mdt\samples-guide.md

This link is broken and I could not find a replacement:
http://msdn.microsoft.com/library/windows/desktop/hh847866.aspx

(admin edit: add metadata for article link)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6fc2393d-6a81-c9c9-87bb-b40b66f26d68
* Version Independent ID: fda85f4c-33ec-d0aa-b793-381d2bebb57d
* Content: [MDT Samples - Microsoft Deployment Toolkit](https://docs.microsoft.com/en-us/sccm/mdt/samples-guide)
* Content Source: [sccm/mdt/samples-guide.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/mdt/samples-guide.md)
* Product: **configuration-manager**
* Technology: **configmgr-osd**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### LINKED ISSUE #1479: Broken Link (troubleshoot-endpoint-client.md)
In the following file: 
SCCMdocs\sccm\protect\deploy-use\troubleshoot-endpoint-client.md 

The following link is broken:
http://www.microsoft.com/en-ph/security_essentials/Support/8c9074b6-1558-4d14-bc39-d294ced11096.aspx

(admin edit: add metadata for link to article)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 940822dc-f344-a2ee-b63d-bc65e2705584
* Version Independent ID: 1f85b90e-b13f-f89f-99e9-d0e11d1fab1e
* Content: [Troubleshooting Windows Defender or Endpoint Protection client - Configuration Manager](https://docs.microsoft.com/en-us/sccm/protect/deploy-use/troubleshoot-endpoint-client)
* Content Source: [sccm/protect/deploy-use/troubleshoot-endpoint-client.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/protect/deploy-use/troubleshoot-endpoint-client.md)
* Product: **configuration-manager**
* Technology: **configmgr-protect**
* GitHub Login: @mestew
* Microsoft Alias: **mstewart**

### LINKED ISSUE #1480: Broken Link (user-driven-installation-developers-guide.md)
In the following file:
C:\Users\chris\SCCMdocs\sccm\mdt\user-driven-installation-developers-guide.md

The following links are broken:
http://www.microsoft.com/visualstudio/products/2010-editions

(admin edit: adding metadata for article link)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5b8ccfad-d059-783f-4384-b5c5c5a4f493
* Version Independent ID: bbc0603d-eb09-66d6-4888-5dfc0d226454
* Content: [User driven installation - Microsoft Deployment Toolkit](https://docs.microsoft.com/en-us/sccm/mdt/user-driven-installation-developers-guide)
* Content Source: [sccm/mdt/user-driven-installation-developers-guide.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/mdt/user-driven-installation-developers-guide.md)
* Product: **configuration-manager**
* Technology: **configmgr-osd**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### LINKED ISSUE #1484: Broken Links (upgrade-to-configuration-manager.md)
The file:
SCCMdocs\sccm\core\servers\deploy\install\upgrade-to-configuration-manager.md

Contains the following broken links:
https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/gg712686
https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/hh846234

(admin edit: add metadata for article link)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 37b73c31-552d-0eda-c477-f44da966acb3
* Version Independent ID: adc49b42-8ca6-702e-d8de-c65e5df69050
* Content: [Upgrade to Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/servers/deploy/install/upgrade-to-configuration-manager)
* Content Source: [sccm/core/servers/deploy/install/upgrade-to-configuration-manager.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/servers/deploy/install/upgrade-to-configuration-manager.md)
* Product: **configuration-manager**
* Technology: **configmgr-other**
* GitHub Login: @mestew
* Microsoft Alias: **mstewart**

### LINKED ISSUE #1485: Broken Link (about-scap.md)
The file:
SCCMdocs\sccm\compliance\plan-design\scap\about-scap.md

Contains the following broken link:
https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/mt228311

(admin edit: add metadata for article link)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7a689124-c901-d2e0-508f-998b39852315
* Version Independent ID: 6f6dc930-ad34-e747-f03f-93c46a3e237c
* Content: [SCAP extensions - Configuration Manager](https://docs.microsoft.com/en-us/sccm/compliance/plan-design/scap/about-scap)
* Content Source: [sccm/compliance/plan-design/scap/about-scap.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/compliance/plan-design/scap/about-scap.md)
* Product: **configuration-manager**
* Technology: **configmgr-compliance**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### PR DESCRIPTION
**Pull request by ChrisKibble**
Fixed number of broken links.

Fixes #1472 
Fixes #1476 
Fixes #1477 
Fixes #1478 
Fixes #1479 
Fixes #1480 
Fixes #1484 
Fixes #1485

### CODE REVIEW

**Comment by PRMerger17** [comment]
@ChrisKibble : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/core/servers/deploy/install/upgrade-to-configuration-manager.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Configuration Manager can't upgrade a primary site that has a database replica f
 
 For more information, see the following articles:  
 
-- System Center 2012 Configuration Manager: [Configure database replicas for management points](https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/hh846234\(v=technet.10))  
+- System Center 2012 Configuration Manager: [Configure database replicas for management points](https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/hh846234\(v=technet.10\))
```
Comment (mestew) [inline sccm/core/servers/deploy/install/upgrade-to-configuration-manager.md]:
```suggestion
- System Center 2012 Configuration Manager: [Configure database replicas for management points](https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/hh846234%28v%3dtechnet.10%29)   
```

File: sccm/core/servers/deploy/install/upgrade-to-configuration-manager.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ Before you disable a task, record the schedule of the task so you can restore it
 
 For more information about site maintenance tasks, see the following articles:  
 
-- System Center 2012 Configuration Manager: [Planning for site operations](https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/gg712686\(v=technet.10))  
+- System Center 2012 Configuration Manager: [Planning for site operations](https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/gg712686\(v=technet.10\))
```
Comment (mestew) [inline sccm/core/servers/deploy/install/upgrade-to-configuration-manager.md]:
```suggestion
- System Center 2012 Configuration Manager: [Planning for site operations](https://docs.microsoft.com/previous-versions/system-center/system-center-2012-R2/gg712686%28v%3technet.10%29)  
```

**Comment by jborsecnik** [comment]
verified URL fixes

**Pull request merged**